libs/geo/src/main/java/org/elasticsearch/geometry/utils/GeographyValidator.java

```diff
@@ -49,10 +52,14 @@
 
     private final boolean ignoreZValue;
 
-    public GeographyValidator(boolean ignoreZValue) {
+    protected GeographyValidator(boolean ignoreZValue) {
         this.ignoreZValue = ignoreZValue;
     }
 
+    public static GeometryValidator get(boolean ignoreZValue) {
```

Comment:
Maybe rename it to "for" so it is more clear that it's just a syntactic sugar for picking one of static instances

Reply:
for is an illegal name, do you have a different suggestion? forValue?

Reply:
I have renamed to #instance.